examples/camera-controller/webrtc-manager/WebRTCManager.cpp

```diff
@@ -81,8 +81,17 @@ CHIP_ERROR WebRTCManager::HandleOffer(uint16_t sessionId, const WebRTCRequestorD
         return CHIP_ERROR_INCORRECT_STATE;
     }
 
-    // Schedule the ProvideICECandidates() call to run asynchronously.
-    DeviceLayer::SystemLayer().ScheduleLambda([this, sessionId]() { ProvideAnswer(sessionId, mLocalDescription); });
+    // Store sessionId for the delayed callback
+    mPendingSessionId = sessionId;
+
+    // Schedule the ProvideAnswer() call to run with a small delay to ensure the response is sent first
```

Comment:
please create an issue and reference it here about us depending on packet ordering here.
UDPs could in theory be dropped and re-ordered so we are essentially saying `300 ms is enough to ensure one packet happens after the other`.

There may be other ways to make things work (e.g. we could wait for RTC before we send the request, so that any order is fine). Relying on sleeps is both slow (because you generally sleep more than really needed just to be sure) and flaky.

Reply:
https://github.com/project-chip/connectedhomeip/issues/39274